src/StaticAnalysisResultsInterchangeFormat/GrammarFiles/SarifFormat.g4

```diff
@@ -619,11 +573,243 @@ region :
     */
     length?;
 
+/**
+    @className {LogicalLocationComponent}
+    @summary {One level (e.g. namespace, function, etc.) of a logical location tree.}
+*/
+logicalLocationComponent:
+    /**
+        @name {Name}
+        @summary {
+        Name of the item specified by this location component.
+        }
+    */
+    name
+
+    /**
+        @name {LocationKind}
+        @summary {
+        The type of item this location refers to.
+        }
+        @remarks {
+        Examples include namespace, class, or function.
+        }
+    */
+    locationKind;
```

Comment:
Actually, there is a problem here: the grammar processor puts a default item named Kind on each class, so no other type can define a member with this name. :) I'll open an issue on this.